forms-flow-web/src/customFormioComponents/FileUpload/fileUploadInputComponent.jsx

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+
+import React, { Component } from 'react';
+
+export default class UploadFile extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            selectedFile: '',
+            name: ""
+        };
+    }
+    onFileChange = (event) => {
+        let file = event.target.files[0];
+        console.log(file);
+        this.setState({ selectedFile: file });
+    };
+
+
+    onFileUpload = () => {
+
+        var timestamp = new Date().getTime();
+        var fileName = timestamp + "-" + this.state.selectedFile.name;
+        console.log(fileName);
+        var formData = new FormData();
+        formData.append("name", fileName);
+        formData.append("File", this.state.selectedFile);
+        console.log("formData---} ", ...formData);
+    
+        axios.post("https://localhost:5000/upload", formData)
+            .then(res => {
+                console.log(res);
```

Comment:
can use success toast here. The user should be able to see the feedback, right?

Reply:
Moving it later